std_msgs/Header header
float64 cam_std_x_dist             # cam_std_x_dist reading
float64 cam_std_x_dist_var    # cam_std_x_dist_var reading
